tests/travis/test_examples.py

```diff
@@ -21,9 +22,16 @@ def test_examples(self):
         proc = subprocess.Popen(["python", script])
         self.assertEqual(proc.wait(), 0, "examples do *NOT* work properly")
 
-    @skipIf(not is_IPython, "IPython is required to run this test")
+    @skipIf(not is_notebook, "Jupyter notebook is required to run this test")
     def test_nbexecuter(self):
         from . import nbexecuter
+
         example_dir = join(dirname(__file__), pardir, pardir, 'examples')
         example_nbconverted = join(example_dir, "glyphs", "glyph.ipynb")
-        nbexecuter.main(example_nbconverted)
+
+        kernel_name = 'python2'
+        pyver = os.environ.get('TRAVIS_PYTHON_VERSION')
+        if pyver in ['3.4', '3.5']:
```

Comment:
This should use something more general, e.g. `pyver.startswith("3.")`, or better `split()`.